test/bin/mdx-test/expect/empty/test-case.md

```diff
@@ -7,7 +7,7 @@ Testing empty blocks
 ```
 
 <!-- $MDX env=x -->
-```foo
+```ocaml
```

Comment:
To avoid having a `env is not allowed for raw blocks` message, which is not the purpose of the test.